apps/client/src/features/rundown/Rundown.tsx

```diff
@@ -110,10 +118,25 @@ export default function Rundown({ data }: RundownProps) {
         newCursor =
           (direction === 'up' ? getLastNormal(rundown, order)?.id : getFirstNormal(rundown, order)?.id) ?? null;
         newIndex = direction === 'up' ? order.length : 0;
+
+        if (jumpBlock && newCursor) {
+          const selected =
+            direction === 'up'
+              ? getRelevantBlockNormal(rundown, order, newCursor)
```

Comment:
suggestion: I think the function name is showing that is unfit as `getRelevantBlock` no longer describes its usage. How about `getPreviousBlock`?